meet-data/pa/URLLIST

```diff
@@ -9,7 +9,6 @@ https://powerliftingaustralia.com/wp-content/uploads/results/2007/NSW_Country_Op
 https://powerliftingaustralia.com/wp-content/uploads/results/2007/NSW_Sydney_Grading_PL_and_BP_Comp_-_12_Feb_2007.html
 https://powerliftingaustralia.com/wp-content/uploads/results/2007/Queensland_PL_and_BP_Open_-_21_Apr_2007.html
 https://powerliftingaustralia.com/wp-content/uploads/results/2007/SE_Challenge_and_Club_Open_-_1_Dec_2007.html
-https://powerliftingaustralia.com/wp-content/uploads/results/2006/2006_World_Championships_Results.html
```

Comment:
Please do not remove anything from the `URLLIST` -- instead, just make `URL` files and it'll be handled normally. The meets removed here seem unrelated to the meets added though -- a bunch of championships from 2006 through 2000. An error?

Reply:
Definitely error, just mean to remove the world champs & Oceania results.